libraries/SrcWrapper/src/stm32/stm32_eeprom.c

```diff
@@ -151,8 +151,17 @@ static uint8_t eeprom_buffer[E2END + 1] __attribute__((aligned(8))) = {0};
   */
 uint8_t eeprom_read_byte(const uint32_t pos)
 {
+#if defined(STM32L0xx)
```

Comment:
Here I guess usage of CMSIS definition should be enough to be more generic.

Reply:
changing STM32L0xx by (FLASH_EEPROM_BASE ||  DATA_EEPROM_BASE)